cypress/integration/blog.spec.js

```diff
@@ -0,0 +1,21 @@
+describe('blog page', function() {
+  it('Check page rendering', function() {
+    cy.visit('http://localhost:8000/blog');
+    cy.contains('With Love from Halo Lab').scrollIntoView({
+      duration: 2000,
+    });
+  });
+  it('check filter', function() {
+    cy.contains('Our Blog').scrollIntoView({
+      duration: 1500,
+    });
+    cy.wait(1000);
+    cy.contains('Agency').click();
+    cy.wait(1000);
+    cy.contains('Case Studies').click();
+    cy.wait(1000);
+    cy.contains('News').click();
+    cy.wait(1000);
+    cy.contains('Agency').click();
```

Comment:
we don't need to test every section, instead, please test one section + click on a particular post. 
after click you should check that title of the clicked element should be equal to title on a blog post page